File: 10.4_MyPhotos/.idea/app/src/main/java/com/example/a55myphotos/model/Photo.java
package com.example.a55myphotos.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Photo implements Serializable {
    private int albumId;
    private int id;
    private String title;
    private String url;
    private String thumbnailUrl;

    public Photo(int albumId, int id, String title, String url, String thumbnailUrl) {
        this.albumId = albumId;
        this.id = id;
        this.title = title;
        this.url = url;
        this.thumbnailUrl = thumbnailUrl;
    }
    public String getTitle(){
        return title;
    }
    public String getUrl(){
        return url;
    }
}

File: MYPHOTOS_Kotlin/app/src/main/java/com/example/myphotos_kotlin/model/Photo.kt
package com.example.myphotos_kotlin.model

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
data class Photo(
    val albumId: Int,
    val id: Int,
    val title: String?,
    val url: String?,
    val thumbnailUrl: String?
) :  Parcelable



